src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationEmailService.java

```diff
@@ -113,46 +93,6 @@ public EmailNotificationDetails sendEmailNotificationToCafcass(CaseData caseData
             .timeStamp(currentDate).build();
     }
 
-    private EmailTemplateVars buildApplicantSolicitorEmail(CaseData caseData, String solicitorName)
```

Comment:
Why is this removed? is it related to SOA at all?

Reply:
Yes for gov notify we were using it so removed it since no longer required